tests/conftest.py

```diff
@@ -194,9 +194,11 @@ def pytest_addoption(parser):
 
 def initial_db_init():
     from airflow.utils import db
+    from airflow.www.app import create_app
 
     db.resetdb()
     db.bootstrap_dagbag()
+    create_app(config={'UPDATE_FAB_PERMS': False})
```

Comment:
If we aren't updating the perms then does creating the app here actually do anything other than take time?

Reply:
Oops. Tests work without this. I probably added this when I was creating `session` table inside the create_app. Thanks for spotting it.

Reply:
I was wrong. It's needed to add some roles. Is there a better way to do it?